test/framework/create_from_yaml.go

```diff
@@ -41,7 +42,11 @@ func NewControllerRuntimeClient() client.Client {
 	})
```

Comment:
```suggestion
	MetricsBindAddress: "0",
	})
```

I think that this change will disable metrics for this controller, which will help us avoid the test flake of
```2021-06-29T13:33:29.011-0400	INFO	exit  	OnError: framework.NewControllerRuntimeClient[create_from_yaml.go:42] unable to start manager: error listening on :8080: listen tcp :8080: bind: address already in use```

Reply:
This change is already in master. Resolving.